.github/workflows/unittests.yml

```
@@ -12,8 +12,8 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - {os: windows-latest, r: "3.6.1"}
-          - {os: macOS-latest,   r: "3.6.1"}
+          - {os: windows-latest, r: "4.0.5"}
+          - {os: macOS-latest,   r: "4.0.5"}
 
```

```suggestion
- {os: ubuntu-20.04, r: "4.0.5"}
```
You can also test it on linux now